src/main/daml/Daml/Finance/Interface/Instrument/Base/Instrument.daml

```diff
@@ -40,6 +40,9 @@ data View = View
     validAsOf : Time
       -- ^ Timestamp as of which the instrument is valid. This usually coincides with the timestamp
       --   of the event that creates the instrument. It usually does not coincide with ledger time.
+      --   This is required for lifecycling of dynamic instruments, in order to keep track of the
```

Comment:
We don't specify what a dynamic instrument is, so I am not sure this is the best place for this comment

Reply:
I have changed it now to make it more general, without using the term _dynamic instrument_.